app/models/manageiq/providers/nuage/inventory/collector/network_manager.rb

```diff
@@ -28,8 +28,8 @@ def security_group(ems_ref)
     security_groups.find { |sg| sg['ID'] == ems_ref }
   end
 
-  def network_group(ems_ref)
-    network_groups.find { |ng| ng['ID'] == ems_ref }
+  def cloud_tenant(ems_ref)
+    cloud_tenants.find { |tenant| tenant['ID'] == ems_ref }
```

Comment:
we could cache it like in target_collection.rb, so we don't do repeated O(n) searches